### PR TITLE
Create web executable

### PR DESCRIPTION
**Pull request by marioEinsis**
- Added main.rs as well as home.rs
- Created cargo.toml
- trunk serve creates prints out AICE Github Dashboard

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "github-dashboard-client"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+wasm-bindgen ="^0.2"
+yew = "^0.19"
```
Comment (msk) [inline Cargo.toml:10]: Please end a text file with a newline. See [Why should text files end with a newline?](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) for justification.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,11 @@
 # github-dashboard-client
 AICE GitHub Dashboard Client
+
+# Usage
+Build and serve with Trunk. Should be running on http://127.0.0.1:8080
+
+```
+trunk build
+trunk serve
+
```
Comment (msk) [inline README.md:10]: This empty line is redundant.

File: src/main.rs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+mod home;
+
+use crate::home::Props;
+use yew;
+
+fn main() {
+    println!("AICE Github Dashboard");
+    let props = Props {};
+    yew::start_app_with_props::<crate::home::Model>(props);
```
Comment (msk) [inline src/main.rs:9]: It is inconsistent to use `crate::home::Props` as `Props` but `crate::home::Model` as `crate::home::Model`. Use both in their short forms (`Props` and `Model`) or use both in their complete forms (`crate::home::Props` and `crate::home::Model`), but do not mix two different forms in a single module.

**Review by msk — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 3 above)
Comment (marioEinsis) [reply src/main.rs:9]: Got it!!

Comment (marioEinsis) [reply src/main.rs:9]: Which form is more preferred? The short form or the complete form?

Comment (msk) [reply src/main.rs:9]: I would keep them short as long as it is clear what they are referring to from the context.

File: Cargo.toml (hunk 1 above)
Comment (msk) [reply Cargo.toml]: index.html has the same problem.

File: src/main.rs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+mod home;
+
+use crate::home::{Model, Props};
+use yew;
+
+fn main() {
+    println!("AICE Github Dashboard");
```
Comment (msk) [inline src/main.rs:7]: Remove this line since an end user won't see this message anyway.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <title>AICE Github Dashboard</title>
```
Comment (msk) [inline index.html:6]: `Github` -> `GitHub`

File: src/home.rs [hunk 6]
```diff
@@ -0,0 +1,28 @@
+use yew::prelude::*;
+
+pub enum Message {}
+
+pub struct Model {}
+#[derive(Properties, Clone, PartialEq)]
```
Comment (msk) [inline src/home.rs:6]: Add a blank line between `struct` definitions for readability.

**Review by msk — CHANGES_REQUESTED** [review]

File: index.html (hunk 5 above)
Comment (msk) [reply index.html:6]: This hasn't been resolved.

Comment (marioEinsis) [reply index.html]: Thank you for catching that. I capitalized the one in home.rs.

**Review by msk — APPROVED** [review]

**Pull request merged**